pyanaconda/core/users.py

```diff
@@ -43,13 +45,26 @@ def crypt_password(password):
     :returns: crypted representation of the original password
     :rtype: str
     """
-    cryptpw = crypt.crypt(password, crypt.METHOD_SHA512)
+    # yescrypt is not supported by Python's crypt module,
+    # so we need to generate the setting ourselves
+    saltchars = string.ascii_letters + string.digits + "./"
```

Comment:
Pylint does not want `string`. That's why the tests fail.

Reply:
> Pylint does not want `string`. That's why the tests fail.

How to solve that?  Simply like this example or does it take more work?

```
import string as _str
…
... _str.ascii_letters ...
```

I'm not a Python expert, as I'm mostly coding in c.

Reply:
I've fixed that in rebased commit.